.github/workflows/reusable-escu-manual-workflow.yml

```diff
@@ -128,7 +130,7 @@ jobs:
         with:
           submodules: recursive
       - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
+        uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
is it correct?

Reply:
I see such branch exists

Reply:
I believe yes, discussed it with @mkolasinski-splunk Can you confirm?